### PR TITLE
EN: Tutorials

### PR DESCRIPTION
**Pull request by DabirSA**
Many tutorial names translated and name format standardised.
Events / Event Chronicle tutorial section name translated
Surrounding Characters tutorial and some dialogue
Event Chronicle tutorial
Autowords tutorial
Good Job tutorial

### CODE REVIEW

File: Tutorial/Dialogue/od_010200381.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+11_voice_NPC_5021_102#0,"""Lately, there's been too many ARKS<br>who don't use Autowords！<br>It's a waste！"""
+text02#0,"""状況に応じたメッセージを出す<br>オートワードは、自身の状況を<br>知らせるのに重要、だというのに……！"""
+text03#0,"""ええい、話している内にも<br>不安が鎌首をもたげてきた！<br>君、今すぐやってくるといい！"""
+text04#0,"""何を、って、決まっているだろう。<br>オートワードの設定だ！<br>クエストクリア時の設定だ！"""
+text05#0,"""Come on, come ooonnn!<br>This shouldn't be difficult!<br>Wipe away my fears!"""
+text00#0,"""Autowords"""
+text01#0,"""Autowords are a chat function which allows<br>your character to speak in specific situations<br>without you having to do anything.<br><br>You can set Autowords with<br>\""System\""→\""Chat Settings\""→\""Autowords\"".<br><br>Register Autowords by entering text<br>in the input fields for each situation.<br><br>If you want more than one Autoword for a situation,<br>press \""+\"" to add an input field."""
+text02#0,"""Greet your allies at the start of a quest,<br>call for help when you're unable to battle;<br>There's all kinds of ways you can use them.<br><br>Have fun with the game<br>using Autowords."""
```
Comment (alama) [inline Tutorial/Dialogue/od_010200381.csv]: this should be text02#1, not text02#0

Comment (DabirSA) [reply Tutorial/Dialogue/od_010200381.csv]: fixed

**Review by alama — APPROVED** [review]

File: UI/ui_tutorial.csv [hunk 2]
```diff
@@ -180,125 +180,125 @@ wh_tl_mp_000245_v#0,"""PSE"""
...
+wh_tl_co_011038_v#0,"""Swords"""
+wh_tl_co_011040_v#0,"""Wired Lances"""
+wh_tl_co_011042_v#0,"""Partisans"""
+wh_tl_od_010200061_v#0,"""Twin Daggers"""
+wh_tl_od_010200071_v#0,"""Double Sabers"""
+wh_tl_od_010200051_v#0,"""Knuckles"""
+wh_tl_co_011050_v#0,"""Gunslashes"""
+wh_tl_co_011052_v#0,"""Assault Rifles"""
+wh_tl_co_011054_v#0,"""Launchers"""
+wh_tl_od_010200131_v#0,"""Twin Machine Guns"""
```
Comment (SynthSy) [inline UI/ui_tutorial.csv:251]: This should be "Twin Machineguns" to fit with the others.

Comment (SynthSy) [reply UI/ui_tutorial.csv:251]: Also, anything with "_v" denotes vita and can be skipped.

File: Tutorial/Window/wh_tl_od_010200441.csv [hunk 3]
```diff
@@ -1,3 +1,3 @@
-name01#0,"""『グッジョブ』について"""
-text01#0,"""グッジョブとは、ほかのプレイヤーに対して<br>ちょっとした応援の気持ちを送る機能です。<br>グッジョブを送ると、<br>相手や自分がFUNを獲得できます。<br><br>キャラクターを選択するか、<br>以下のメニューから「グッジョブを送る」を<br>選ぶと送信出来ます。<br>・フレンドリスト<br>・チームメンバー一覧<br>・パーティーメンバー<br>・周囲のキャラクター<br>・コミュニケーション履歴<br>・アークスサーチ"""
-text02#0,"""グッジョブはコメント無しでも<br>送信できますが、コメントを付けると<br>より多くのFUNを獲得できます。<br><br>同じプレイヤーへのグッジョブで<br>FUNを獲得できるのは、1日1回までです。"""
+name01#0,"""Good Jobs"""
```
Comment (SynthSy) [inline Tutorial/Window/wh_tl_od_010200441.csv:1]: Do not omit について as this means "About".
Proper title should be "About "Good Jobs".

File: Tutorial/Window/wh_tl_od_010200381.csv [hunk 4]
```diff
@@ -1,3 +1,3 @@
-name01#0,"""『オートワード』について"""
-text01#0,"""オートワードとは、プレイヤーによる操作無しで、<br>特定の状況になると自動的に発言する<br>チャット機能です。<br><br>「システム」→「チャット設定」→「オートワード」で<br>設定することが出来ます。<br><br>状況別の入力欄を選び、<br>テキストを入力すると登録できます。<br><br>1つの状況に複数のオートワードを<br>登録したい場合は<br>「＋」を選んで入力欄を増やしましょう。"""
-text02#0,"""クエスト開始時に挨拶をしたり、<br>戦闘不能になった際に助けを呼んだり<br>使用方法は様々です。<br><br>オートワードを活用して<br>楽しくゲームをプレイしましょう。"""
+name01#0,"""Autowords"""
```
Comment (SynthSy) [inline Tutorial/Window/wh_tl_od_010200381.csv:1]: Do not omit について as this means "About".
Proper title should be About "Autowords".

File: Tutorial/Window/wh_tl_co_040020.csv [hunk 5]
```diff
@@ -1,2 +1,2 @@
-name01#0,"""『周囲のキャラクター』について"""
-text01#0,"""「クイックメニュー」から<br>「周囲のキャラクター」を使用することで<br>自分の周りにいるキャラクターをリストで<br>表示することができます。<br><br>リストの中からキャラクターを選択することで<br>キャラクター情報やアークスカードの確認、<br>パーティへの勧誘を行うことが可能です。"""
+name01#0,"""Surrounding Characters"""
```
Comment (SynthSy) [inline Tutorial/Window/wh_tl_co_040020.csv:1]: Do not omit について as this means "About".
Proper title should be About "Nearby Characters".

**Comment by SynthSy** [comment]
Once these changes are made, resubmit your PR please.

**Pull request merged**